api/client/builder/types.go

```diff
@@ -1008,12 +1009,13 @@ func (ehr *ExecHeaderResponseDeneb) ToProto() (*eth.SignedBuilderBidDeneb, error
 }
 
 // ToProto creates a BuilderBidDeneb Proto from BuilderBidDeneb.
-func (bb *BuilderBidDeneb) ToProto() (*eth.BuilderBidDeneb, error) {
+func (bb *BuilderBidDeneb) ToProto(s types.Slot) (*eth.BuilderBidDeneb, error) {
```

Comment:
Why not have this be the max blob count instead of the slot? That would be a better abstraction since this method doesn't actually care about the slot, it only needs it to derive the max blob count